### ISSUE TITLE
Broken keyboard navigation

### ISSUE THREAD

**Issue by mdafanasev** [postcss/postcss.org#247]
Keyboard navigation order is broken.
Expected behavior:
Pressing the Tab key first time moves focus to the first navigation element.
Actual behavior:
Visually nothing.
There are two problems here:
1. Gitter chat window is inserted before other page elements (outside body!). It should be hidden by "display: none", but it use "translateX(110%)". So, elements inside this window get focus before navigation links.
2. Navigation component is placed after page content, so it's elements get focus only after other links.

**Comment by marcustisater** [comment]
Great catch @mdafanasev. Accessibility is important and this should be fixed.

**Comment by Aaronmoi** [comment]
Fix

**Comment by Aaronmoi** [comment]
-started.md
